inst/examples/build-a-box/R/mod-global-controls.R

```diff
@@ -0,0 +1,118 @@
+ui_global_controls <- function(id) {
+  ns <- shiny::NS(id)
+
+  tagList(
+    layout_columns(
+      class = "align-items-end",
+      selectizeInput(
+        ns("theme_style"),
+        "Theme style",
+        list(
+          "All" = "all",
+          "Semantic Colors" = list(
+            "Background" = "semantic-bg",
+            "Text" = "semantic-fg"
+          ),
+          "Theme Colors" = list(
+            "Background" = "colors-bg",
+            "Text" = "colors-fg"
+          ),
+          "Vibrant" = list(
+            "Gradient" = "gradient"
+          )
+        )
+      ),
+      div(
+        class = "btn-group",
+        role = "group",
+        "aria-label" = "Theme actions",
+        shuffleButton(ns("random_theme"), "Theme"),
+        shuffleButton(ns("random_stat"), "Stats"),
+        actionButton(ns("reset_theme"), "Reset")
+      )
+    ),
+    layout_columns(
+      class = "align-items-start",
+      div(
+        radioButtons(
+          ns("showcase_item"),
+          "Showcase Item",
+          choices = c("Plot", "Icon"),
+          inline = TRUE
+        ),
+        conditionalPanel(
+          "input.showcase_item == 'icon'",
+          ns = ns,
+          shuffleButton(ns("random_icon"), "Icons")
+        )
+      ),
+      radioButtons(
+        ns("showcase_layout"),
+        "Showcase Layout",
+        choices = c("Left center", "Top right", "Bottom"),
+        inline = TRUE
+      )
+    )
+  )
```

Comment:
I think this (meaning the global controls tab) UI, in general, could do a little more to guide the user experience. Here is a couple ideas:

1. Provide a button towards the bottom that says something like "Customize 1st value box". Without some visual cue like this I think the use might be left wondering what to do after exploring global controls.
2. Sort of in line with (1), perhaps include a tooltip on the theme dropdown that explains how this is a "high-level" theme control, and "lower-level" controls can be accessed on a particular value box pages.

Reply:
Great points. I added a "?" box next to "Value box settings" with a popover explaining how to use the settings panel.